cpp_linter/rest_api/github_api.py

```diff
@@ -66,6 +76,66 @@ def set_exit_code(
             checks_failed, format_checks_failed, tidy_checks_failed
         )
 
+    @staticmethod
+    def _rate_limit_exceeded(
+        response_headers: Union[
+            requests.structures.CaseInsensitiveDict, Dict[str, Any]
+        ],
+    ):
+        logger.error("RATE LIMIT EXCEEDED!")
+        if "x-ratelimit-reset" in response_headers:
+            reset_epoch = time.gmtime(
+                int(cast(str, response_headers.get("x-ratelimit-reset")))
+            )
+            logger.error(
+                "Github REST API rate limit resets on %s",
+                time.strftime("%d %B %Y %H:%M +0000", reset_epoch),
+            )
```

Comment:
I should probably exit early with `sys.exit(1)` to highlight the rate limit failure. Currently, the action will still pass with these logged errors, but that would produce a LGTM result when the rate limit is exceeded.

Reply:
done